tests/conftest.py

```diff
@@ -323,10 +323,31 @@ async def read_sorted_messages(
 @pytest.fixture
 def assert_no_more_messages(websocket):
     """Assert that there are no more messages on the websocket."""
-    async def assert_no_more_messages(timeout: float | None):
-        with pytest.raises(asyncio.TimeoutError):
-            await asyncio.wait_for(read_JSON_message(websocket),
-                                   timeout=timeout)
+    async def assert_no_more_messages():
```

Comment:
I am not sure it guarantees much for network events for example

Reply:
neither does a timeout of a second

Reply:
But we do need to check there are no more messages are coming. I guess this approach covers the risks it is expected to cover.